code/model/MultiForm.php

```diff
@@ -176,6 +176,15 @@ public function getController() {
 	}
 
 	/**
+	 * Returns the get_var to the template engine
+	 *
+	 * @return string
+	 */
+	public function get_var() {
```

Comment:
I can adjust it to be consistent with the naming which would be "getGetVar" - pretty weird name. What do you think @tractorcow ?